tests/framework/nodeplacement.go

```diff
@@ -53,11 +53,36 @@ func (f *Framework) TestNodePlacementValues() sdkapi.NodePlacement {
 }
 
 // PodSpecHasTestNodePlacementValues compares if the pod spec has the set of node placement values defined for testing purposes
-func (f *Framework) PodSpecHasTestNodePlacementValues(podSpec v1.PodSpec) bool {
+func (f *Framework) PodSpecHasTestNodePlacementValues(name string, podSpec v1.PodSpec) bool {
 	if !reflect.DeepEqual(podSpec.NodeSelector, nodeSelectorTestValue) {
 		fmt.Printf("mismatched nodeSelectors, podSpec:\n%v\nExpected:\n%v\n", podSpec.NodeSelector, nodeSelectorTestValue)
 		return false
 	}
+	for _, deploymentname := range []string{"cdi-apiserver", "cdi-deployment", "cdi-uploadproxy"} {
```

Comment:
Don't think you need this loop at all. you pass in the deploymentName, so this is essentially creating an affinity with a particular deployment name, and you can just set it to the parameter.

Reply:
I add this loop because in import_test.go also use this `PodSpecHasTestNodePlacementValues`, and import Pod dosen't have that affinity.
![image](https://github.com/kubevirt/containerized-data-importer/assets/49519342/313e2597-bd79-4978-8457-6acf7d6e6cd6)
I think I need revert this `PodSpecHasTestNodePlacementValues`,and add `CheckAntiAffinity(...)CheckTolerations(...)` in operator_test.go just like you said to replace `PodSpecHasTestNodePlacementValues` in operator_test.go.

Reply:
Btw, I see that the original code for checking tolerations was not changed, but you can make it better by doing something like:
```go
Expect(podSpec.Tolerations).To(ContainElmement(tolerationsTestValue[0])
```
There is a gomega matcher that allows you to check if a slice contains an element. We haven't done that properly in the past and when I see one I always try to fix it.